documentation/book/con-how-the-topic-operator-works.adoc

```diff
@@ -0,0 +1,21 @@
+[id='how-the-topic-operator-works-{context}']
+= How the Topic Operator works
+
+A fundamental problem that the operator has to solve is that there is no single source of truth: 
+Both the `KafkaTopic` resource and the topic within Kafka can be modified independently of the operator.
+Complicating this, the Topic Operator might not always be able to observe changes at each end in real time (for example, the operator might be down).
+ 
+To resolve this, the operator maintains its own private copy of the information about each topic. 
+When a change happens either in the Kafka cluster, or in {ProductPlatformName}, it looks at both the state of the other system, and at its private copy in order to determine what needs to change to keep everything in sync.  
+The same thing happens whenever the operator starts, and periodically while it is running.
+
+For example, suppose the Topic Operator is not running, and a `KafkaTopic` "my-topic" gets created.
```

Comment:
please use monospace for filenames, that is  "my-topic" to `my-topic`.

http://strimzi.io/contributing/guide/#formatted-text-reference